ShelterModule/Models/Announcements/Announcement.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Database.Entities;
+
+namespace ShelterModule.Models.Announcements;
+
+public sealed class Announcement
+{
+    public Guid Id { get; init; }
+    public required Guid AuthorId { get; init; }
+    public required Guid PetId { get; init; }
+    public required string Title { get; init; } = null!;
+    public required string Description { get; init; } = null!;
+    public required DateTime CreationDate { get; init; }
+    public DateTime? ClosingDate { get; init; }
+    public required int Status { get; init; }
```

Comment:
IDK if this should be an `int`. `enum` would be more readable. If specification says so, we can accept `int` in requests and return `int` in responses, but internally it would be nicer to use an enum. Actually, I think if we have an `enum` property in DTO classes (Data Transfer Objects, so request and response models), it's converted to `int` by default.